examples/master.js

```diff
@@ -356,7 +360,30 @@ async function getIceServersWithCaching(formValues) {
 
     // Add the TURN servers unless it is disabled
     if (!formValues.natTraversalDisabled && !formValues.forceSTUN && formValues.sendRelayCandidates) {
-        const [turnServers, turnServerExpiryTsMillis] = await master.channelHelper.fetchTurnServers();
+        let [turnServers, turnServerExpiryTsMillis] = await master.channelHelper.fetchTurnServers();
+
+        if (!formValues.turnWithUdp || !formValues.turnsWithUdp || !formValues.turnsWithTcp) {
+            turnServers = turnServers.map((config) => {
+                return {
+                    urls: config.urls.filter((url) => {
+                        if (url.startsWith('turn:') && url.endsWith('?transport=udp')) {
+                            return formValues.turnWithUdp;
+                        } else if (url.startsWith('turns:') && url.endsWith('?transport=udp')) {
+                            return formValues.turnsWithUdp;
+                        } else if (url.startsWith('turns:') && url.endsWith('?transport=tcp')) {
+                            return formValues.turnsWithTcp;
+                        }
+                    }),
+                    username: config.username,
+                    credential: config.credential,
+                };
+            });
+        }
+
+        if (formValues.oneTurnServerSetOnly) {
```

Comment:
Might want to update the screenshot in the description, I was confused why it was randomizing them instead of using the first one as it shows there

Reply:
Updated the screenshot - I originally had it as the first one `[0]`, but then decided that it was probably better to choose one at random to help balance out the load.